File: Chef and Bird farm.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,y,z;
        scanf("%d%d%d",&x,&y,&z);
        if(z%x==0&&z%y==0)
        printf("ANY\n");
        else if(z%x==0)
        printf("CHICKEN\n");
        else if(z%y==0)
        printf("DUCK\n");
        else
        printf("NONE\n");
    }
}

File: Chef and Bird farm.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int x=sc.nextInt();
            int y=sc.nextInt();
            int z=sc.nextInt();
            if(z%x==0&&z%y==0)
            System.out.println("ANY");
            else if(z%x==0)
            System.out.println("CHICKEN");
            else if(z%y==0)
            System.out.println("DUCK");
            else
            System.out.println("NONE");
        }
    }
}